AstSemantics.md

```diff
@@ -553,7 +553,10 @@ outside the range which rounds to an integer in range) traps.
   * `select`: a ternary operator with a boolean (i32) condition and two
     additional operands, which must have the same type as each other. `select`
     returns the the first of these two operands if the condition operand is
-    non-zero, or the second otherwise.
+    non-zero, or the second otherwise. The order of evaluation is the left
```

Comment:
I think the order is already specified, where it says `All nodes other than control flow constructs need to evaluate their child nodes in the order they appear in the serialized AST.` I feel that specifying it here redundantly might be confusing (people might think, why is this specified here, and why don't other ops say their order of evaluation).

Perhaps remove the first added sentence, and keep the last, replacing "therefore" with "note that"?